.devcontainer/devcontainer.json

```diff
@@ -4,7 +4,12 @@
 	"name": "devcontainers-ci",
 	"dockerFile": "Dockerfile",
 	"build": {
-		"cacheFrom": "ghcr.io/devcontainers/ci-devcontainer:latest"
+		"cacheFrom": "ghcr.io/devcontainers/ci-devcontainer:latest",
+		"args": {
+			// This is a temporary workaround when developing from host -> remote host -> devcontainer
+            // see: https://github.com/microsoft/vscode-remote-release/issues/7958
```

Comment:
It was mentioned that updating Docker helped. Is this still needed?

Reply:
I was unable to get the devcontainer to build without this even after updating to the latest docker extension for vscode so I have left this as is.